debian/build.go

```diff
@@ -275,6 +280,10 @@ func getReleaseDownloadLinkBase(v version) (string, error) {
 	return fmt.Sprintf("https://dl.k8s.io/v%s", v.Version), nil
 }
 
+func getCriToolsDownloadLinkBase(v version) (string, error) {
+	return fmt.Sprintf("https://github.com/kubernetes-incubator/cri-tools/releases/download/v%s", v.Version), nil
```

Comment:
You can/should include all of this logic in `debian/rules` as you have to do that kind of golang templates there anyway

Reply:
this is the thing that populates .DownloadLinkBase in `debian/rules`...i think i'm misunderstanding something

Reply:
Yes I know, but I meant that you can move all of the logic into `debian/rules` because it's the same URL for all cases (you don't have two different `getCriToolsDownloadLinkBase` funcs). You have access to `{{ .Version }}` in debian rules as well, and then it's more similar to the rpm packaging and less code. Makes sense?

Reply:
oh, you want to have `GetDownloadLinkBase` for the `[]build` be `nil` in the cri-tools case and replace `{{ .DownloadLinkBase }}` in debian/rules with the hardcoded github path?